src/cdk/schematics/utils/version-agnostic-typescript.ts

```diff
@@ -36,4 +36,4 @@ try {
   }
 }
 
-export {ts};
+export {ts, typescript};
```

Comment:
Why do we need to export the `typescript` thing here? Shouldn't everything just use the agnostic TypeScript version which is also typed?

Reply:
My understanding is that the `ts` and the `typescript` refer to two different namespaces. `ts` didn't have `SourceFile` which we needed.

Reply:
The `ts` object should be typed with `typeof typescript` but I could see that this one omits the actual  types as it's not declared as namespace. Though it feels ambiguous to export `ts` and `typescript`.

Why can't we just rely on the implicit type from `ts.createSourceFile`? I don't bother too much, but there is definitely some ambiguity now. If you want to explicitly type it, how about just importing `typescript` there and type it from there. We just need to use the agnostic TS version at _runtime_

Reply:
We can't let it infer the type from `createSourceFile`, because Bazel was throwing an error about a dependency being brought in implicitly.

Reply:
hmm. There should be no change for dependencies with the current changes. I'm okay getting it in like that. I can just explore cleaning up the ambiguous export in a follow-up then.